ios/Podfile.lock

```diff
@@ -722,7 +722,7 @@ DEPENDENCIES:
   - "RNCClipboard (from `../node_modules/@react-native-community/clipboard`)"
   - "RNCPicker (from `../node_modules/@react-native-picker/picker`)"
   - "RNDateTimePicker (from `../node_modules/@react-native-community/datetimepicker`)"
-  - "RNFastImage (from `../node_modules/@pieter-pot/react-native-fast-image`)"
+  - RNFastImage (from `../node_modules/react-native-fast-image`)
```

Comment:
Out of curiosity, why are some of these wrapped in "" and others are not? No difference?

Reply:
Yeah there shouldn't be any difference at all there - I think its simply a difference between XCode versions or something like that, but doesn't actually effect the pod in anyway :)